### PR TITLE
compute_cve

### LINKED ISSUE #91: incorporate different cve computation methods to compute_cve
Based on discussions in #87:
- We load everything from the `fast_cve` module to the `functions` module.
- Specifically, `compute_cve` should take `DO`, `muD`, and `method` as inputs and return a `cve DO`. It calls `cve_method` through the `method` and passes `DO` and `muD` to compute the cve based on the `global_tth`. We then interpolate the cve to the original grid.

### PR DESCRIPTION
**Pull request by yucongalicechen**
- closes #91 
- continue from #87. I thought it'd be cleaner to start from a new branch
- addressed comments: use a more understandable name than abdo, add cve_methods to docstring.
- I haven't changed the default method to None. When we load the args into headers we will write the method, so if we want to have default method to None, we need a function to change args.method=None to args.method="polynomial_interpolation". Is it easier if we just have the default value as "polynomial_interpolation"?

### CODE REVIEW

**Review by sbillinge — COMMENTED** [review]
nicely done!  There is one inline comment.

Also, codecov is failing.  We need some tests for when things go wrong.  This would be in a separate test called `test_compute_cve_bad`.  These tests are not strictly necessary but definitely make the code better so let's do them while we are at it.

Let's also make an issue to replace the interpolation step with one using diffraction objects if that ever gets implemented.  I think there should be an issue in diffpy.utils for that so link to that issue.

**Comment by sbillinge** [comment]
@yucongalicechen one more thing, we need a news for this.

File: src/diffpy/labpdfproc/labpdfprocapp.py [hunk 1]
```diff
@@ -72,6 +72,13 @@ def get_args(override_cli_inputs=None):
         action="store_true",
         help="Outputs will not overwrite existing file unless --force is specified.",
     )
+    p.add_argument(
+        "-m",
+        "--method",
+        help=f"The method for computing absorption correction. Allowed methods: {*CVE_METHODS, }. "
+        f"Default method is polynomial interpolation if not specified. ",
```
[the remark this answers is not in the document]
Comment (yucongalicechen) [reply src/diffpy/labpdfproc/labpdfprocapp.py:79]: black seems to automatically adjust the indentation for all help messages and aligns them at help. I can try to add a paranthesis around the messages to make sure they line up with the lines above, if that's helpful?

[the remark this answers is not in the document]
Comment (sbillinge) [reply src/diffpy/labpdfproc/labpdfprocapp.py]: No, we have to fix it. Did you commit to main by mistake? That could cause this behavior

[the remark this answers is not in the document]
Comment (yucongalicechen) [reply src/diffpy/labpdfproc/labpdfprocapp.py]: Okay, I managed to resolve conflicts. I think it just needs me to merge from main again.

**Comment by sbillinge** [comment]
Nice, thanks!

**Pull request merged**